portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/categories/CategoriesOOTB.testcase

```diff
@@ -0,0 +1,133 @@
+@component-name = "portal-categories"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Content Dashboard";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		JSONGroup.addGroup(groupName = "Test Site Name");
+
+		Navigator.openURL();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+		}
+	}
+
+	@description = "When the user clicks on Save and Add a new one, the category is saved"
+	@priority = "5"
+	test AddCategoryUsingSaveAndAddButton {
+		property portal.acceptance = "true";
+
+		Category.openCategoriesAdmin(siteURLKey = "global");
+
+		Category.AddCategoryWithSaveAndAddButton(
+			categoryName = "Category 1",
+			vocabularyName = "Topic");
+
+		AssertTextPresent(value1 = "Category 1");
+	}
+
+	@description = "When the site administrator (not Global) accesses to the Categories administration application, the existing Global vocabularies appear as locked and it's not possible to edit them"
+	@priority = "5"
+	test GlobalCategoriesNotEditable {
+		property portal.acceptance = "true";
+		JSONCategory.addCategory(
+			groupName = "Global",
+			title = "Topic 1",
+			vocabularyName = "Topic");
+
+		Category.openCategoriesAdmin(siteURLKey = "test-site-name");
+
+		AssertElementPresent(locator1 = "Categories#LOCKED_VOCABULARY");
+
+		Category.viewCP(
+			categoryName = "Topic 1",
+			vocabularyName = "Topic");
+
+		AssertElementNotPresent(locator1 = "Categories#ADD_CATEGORY_BUTTON");
+
+		Category.openCategoriesAdmin(siteURLKey = "global");
+
+		Category.deleteCategoriesCP(vocabularyName = "Topic 1");
```

Comment:
You are passing a vocabulary as a parameter, so you can not use the "deleteCategoriesCP" method ...
replace this kind of thing to:

`		Category.deleteByIconCP(
			categoryName = "Topic 1",
			vocabularyName = "Topic");
`